arborator/main.py

```diff
@@ -191,6 +192,7 @@ def process_data(group_files,id_col,group_col,thresholds,outlier_thresh,method,n
 def process_group(group_id,output_files,id_col,group_col,thresholds,outlier_thresh,method,min_members=2):
     (allele_map, df) = process_profile(output_files['profile'], column_mapping={})
     l, p = convert_profiles(df)
+    l = [str(x) for x in l]
```

Comment:
As mentioned before here: https://github.com/phac-nml/arborator/pull/19/files#r2240916843

Reply:
Already addressed in numeric id pull https://github.com/phac-nml/arborator/pull/19